Makefile

```diff
@@ -86,6 +87,18 @@ V ?= $(or $(VERBOSE),0)
 Q = $(if $(filter 1,$V),,@)
 H = $(shell printf "\033[34;1m=>\033[0m")
 
+# To build Pilot, Mixer and CA with debugger information, use DEBUG=1 when invoking make
+DEBUG ?= 0
+ifeq (${DEBUG},1)
+	DEBUG_MODULES = 1
+    $(info $(H) Build with debugger information)
+else ifeq ($(DEBUG),0)
+	DEBUG_MODULES = 0
+else
+    $(error DEBUG has a default value of 0 for regular build and 1 for debugging)
+endif
+export DEBUG_MODULES
```

Comment:
do we need both `DEBUG` and `DEBUG_MODULES` ? they seem to have the same value ?
it's also an odd name (it seems like it would be a list of modules)

Reply:
you are right. It came from earlier version. I'll remove it.